apps/e2e/tests/backend/endpoints/api/v1/auth-flows.test.ts

```diff
@@ -0,0 +1,109 @@
+import { it } from "../../../../helpers";
+import { Auth, ContactChannels, backendContext, niceBackendFetch } from "../../../backend-helpers";
+
+it("signs in with OTP, disable used for auth, then should not be able to sign in again", async ({ expect }) => {
```

Comment:
test names should be legible with the title of the file as a prefix; "endpoints/api/v1/auth-flows signs in with OTP, disable used for auth, then should not be able to sign in again" is a confusing (and grammatically wrong) sentence

(this also applies to all the other tests)